### PR TITLE
Feat: Update frontend display value for Internship to Internship/Traineeship

### PR DESCRIPTION
**Pull request by witxhhaven**
[story link](https://www.pivotaltracker.com/n/projects/2434214/stories/172194713)

feat: amend values and add logic by Raphael, updated the test.

### CODE REVIEW

File: packages/constants/src/mcf.ts [hunk 1]
```diff
@@ -49,6 +49,9 @@ const mapMsfToMcfEmploymentTypes = () => {
       if (employmentType.ilpDescription === 'Contract Basis') {
         return {id: parseInt(employmentType.ilpId, 10), employmentType: 'Contract'};
       }
+      if (employmentType.ilpDescription === 'Internship') {
```
Comment (shonoru) [inline packages/constants/src/mcf.ts:52]: Please add story link in comment, to explain change and stating this as special case

Comment (witxhhaven) [reply packages/constants/src/mcf.ts:52]: okie

Comment (shonoru) [reply packages/constants/src/mcf.ts:52]:
```
// NOTE: https://www.pivotaltracker.com/n/projects/2434214/stories/172194713
```

Comment (shonoru) [reply packages/constants/src/mcf.ts:52]: Even butter,

```
// NOTE: part of the support for the recently announced SGUnitedTraineeship, https://www.pivotaltracker.com/n/projects/2434214/stories/172194713
```

Comment (shonoru) [reply packages/constants/src/mcf.ts:52]: Nit.

**Pull request merged**